.github/pull_request_template.md

```diff
@@ -5,12 +5,12 @@
 ...
 
 ### Checklist before you submit for review
-- [ ] Make sure there is a PR in the CNDB project updating the Converged Cassandra version
+- [ ] Make sure there is a PR in the CNDB project updating the Converged Cassandra version if your PR is changing the API
 - [ ] Use `NoSpamLogger` for log lines that may appear frequently in the logs
-- [ ] Verify test results on Butler
-- [ ] Test coverage for new/modified code is > 80%
+- [ ] Verify Butler and Sonar checks are passing
 - [ ] Proper code formatting
 - [ ] Proper title for each commit staring with the project-issue number, like CNDB-1234
 - [ ] Each commit has a meaningful description
-- [ ] Each commit is not very long and contains related changes
-- [ ] Renames, moves and reformatting are in distinct commits
\ No newline at end of file
+- [ ] Each commit contains related changes
+- [ ] Renames, moves and reformatting are in distinct commits
+- [ ] Avoid starting nodes in new tests (jvm dtests, CQLTester) if possible in favor of implementing pure unit tests
```

Comment:
I don't quite agree with this: in a complex distributed system, integration tests are fundamental and should not be discouraged _even if unit tests exist_.

Reply:
though starting nodes over and over again should be avoided - if we are adding a test case to the test class which already has some shared containers running - that's fine; however we should design tests having the testing time in mind;

Reply:
Agreed, but this is more of a general "know how to write efficient tests" issue?